[2025-02-05 18:42:54,612][utils.py:44][ERROR] Error creating raw table: 'Update_time'
[2025-02-05 18:43:03,835][utils.py:44][ERROR] Error creating raw table: 'Update_time'
[2025-02-05 18:44:02,918][utils.py:44][ERROR] Error creating raw table: 'Update_time'
[2025-02-05 18:44:06,483][utils.py:44][ERROR] Error creating raw table: 'Update_time'
